.github/workflows/cpp.yml

```diff
@@ -97,3 +97,4 @@ jobs:
     - name: Test
       run: |
         "${GITHUB_WORKSPACE}/bin/bazel" build //...
+        "${GITHUB_WORKSPACE}/bin/bazel" build //example:http_wasm_example.wasm
```

Comment:
That's already included in `//...`, isn't it?

Reply:
This is from the envoy rule, which adds manual tag to avoid build with wildcard path.  We can remove it here and add it in upstream by the customizing envoy_wasm_cc_rule.